src/pip/_internal/req/req_install.py

```diff
@@ -571,8 +571,30 @@ def get_pep_518_info(self):
         if os.path.isfile(self.pyproject_toml):
```

Comment:
How about taking the opportunity to simplify the function by getting the non PEP 518 case out of the way first:
```python
        if not os.path.isfile(self.pyproject_toml):
	    return (['setuptools', 'wheel'], False) 
```

Reply:
Yeps. Good idea.